_components/pages/History.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from "react";
+
+export default function History() {
+  return (
+    <div className="overflow-hidden landing">
+      <div className="position-relative">
+        <img
+          src="./img/backgrounds/history-orange.svg"
```

Comment:
For the image sources to work on the preview site, you should apply a prefix to these paths. That goes something like this:

```tsx
import { usePathPrefix } from '@redocly/developer-portal/ui';

export default function History() {
  const prefix = usePathPrefix()
  return (
    <div className="overflow-hidden landing">
      <div className="position-relative">
        <img
          src={prefix + "/img/backgrounds/history-orange.svg"}